### PR TITLE
Improve README and fix compatibility with latest version

### PR DESCRIPTION
**Pull request by xsellier**
Hey,

Thank yo ufor that wonderful application.
I'm also a Starlink user and I wanted to install starlink-monitor on a abare-metal reaspberry-i, so I added instructions to the README.md file to do it.

I noticed there was a compatibility issue with the latest firmware, so I tweaked the `starlink.py` to make it work.

Hope you'll like the changes

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,6 +7,7 @@ ENV STARLINK_HISTORY_REFRESH_SECS=30
 ENV BROADCAST_RATE_SECS=3
 ENV SPEEDTEST_REFRESH_MINS=30
 ENV BUFFER_SIZE_HOURS=72
+ENV SERVER_HTTP_PORT=3000
```
Comment (dbjohnson) [inline Dockerfile]: to match current default, this should be port 80; 3000 is what the container exposes

File: Dockerfile.arm32 [hunk 2]
```diff
@@ -12,6 +12,7 @@ ENV STARLINK_HISTORY_REFRESH_SECS=30
 ENV BROADCAST_RATE_SECS=3
 ENV SPEEDTEST_REFRESH_MINS=30
 ENV BUFFER_SIZE_HOURS=72
+ENV SERVER_HTTP_PORT=3000
```
Comment (dbjohnson) [inline Dockerfile.arm32]: to match current default, this should be port 80; 3000 is what the container exposes

File: README.md [hunk 3]
```diff
@@ -43,15 +43,38 @@ docker run -d -p 3000:80 \
 
 ## Build for development
 
-```
+```bash
 docker-compose build && docker-compose up -d
 ```
 
+## Deploy on bare-metal Raspberry pi
```
Comment (dbjohnson) [inline README.md]:
```suggestion
## Deploy on bare-metal Raspberry pi (without using Docker)
```

File: README.md [hunk 4]
```diff
@@ -43,15 +43,38 @@ docker run -d -p 3000:80 \
...
+sudo systemctl enable starlink.service
+sudo systemctl start starlink.service
+```
+
 
 ## Environment variables
 
 | env                             | description                              | default            |
 | ------------------------------- | ---------------------------------------- | ------------------ |
+| `SERVER_HTTP_PORT`              | starlink monitoring server port          | 1337               |
```
Comment (dbjohnson) [inline README.md]:
```suggestion
| `SERVER_HTTP_PORT`              | starlink monitoring server port          | 80               |
```

File: app/app.py [hunk 5]
```diff
@@ -8,37 +9,37 @@
 
 
 app = Flask(__name__)
-socketio = SocketIO(app, async_mode='threading')
+socketio = SocketIO(app)
```
Comment (dbjohnson) [inline app/app.py]: Can you describe why this change is necessary?

**Review by dbjohnson — CHANGES_REQUESTED** [review]
Hey @xsellier - thanks so much!  `starlink-monitor`'s first collaborator PR 🎉 !!

I appreciate adding the instructions for how to deploy on pi sans docker.  I had one question re: the threading change (just curious why) and two minor blockers:

1) the default HTTP port should stay at 80 - in the usual docker config, this is only exposed in container, and remapped to 3000 outside.  I'm fine with changing it to something other than 80 to support this bare install case, but then the README examples would need to be updated to match
2) All the white space changes need to be reverted.  I'm not super pedantic about this stuff, but I like to follow PEP8 ([4 space tabs](https://www.python.org/dev/peps/pep-0008/#indentation)).  More generally, most of these files have no functional change. 
 Probably just an auto-formatting issue, hopefully easy to back out :)

Thanks again for the PR - will be glad to merge if you are up for correcting those two issues!

File: README.md [hunk 6]
```diff
@@ -11,7 +11,7 @@ Background image credit: [u/johnkphotos](https://www.reddit.com/r/space/comments
 
 ## Quickstart
 
-```
+```bash
```
Comment (dbjohnson) [inline README.md:14]: 👍

**Comment by xsellier** [comment]
Thank you for the review!

File: Dockerfile [hunk 7]
```diff
@@ -7,6 +7,7 @@ ENV STARLINK_HISTORY_REFRESH_SECS=30
 ENV BROADCAST_RATE_SECS=3
 ENV SPEEDTEST_REFRESH_MINS=30
 ENV BUFFER_SIZE_HOURS=72
+ENV SERVER_HTTP_PORT=80
```
Comment (dbjohnson) [inline Dockerfile:10]: 👍

**Review by dbjohnson — CHANGES_REQUESTED** [review]
Thanks for updating the port!  Please see other comments from the initial review, particular the whitespace issue.

**Comment by xsellier** [comment]
It's ok, I'm not gonna change the whitespaces, soryr for bothering

**Comment by dbjohnson** [comment]
It's no bother - I sincerely appreciate the interest and offer for help!  

But, PRs shouldn't include reformatting to match individual stylistic preferences / dev setups.  It makes the functional changes hard to differentiate.

**Pull request closed without merging**